java/src/jmri/jmrit/display/layoutEditor/MultiSensorIconFrame.java

```diff
@@ -164,11 +176,11 @@ public String toString() {
         Entry(JPanel self, JmriJFrame frame, String name) {
             this.self = self;
             this.setLayout(new FlowLayout());
-            this.add(new JLabel("Sensor:"));
+            this.add(new JLabel(Bundle.getMessage("BeanNameSensor") + ":"));
```

Comment:
These colons need to be included in the .properties files, and not left here outside the translation. At least French and Japanese (both languages we have partial support for) use a space between the leading word and the colon (unlike English or German).